src/app/application.cpp

```diff
@@ -645,41 +647,52 @@ int Application::exec(const QStringList &params)
     Net::DownloadManager::initInstance();
     IconProvider::initInstance();
 
-    try
-    {
-        BitTorrent::Session::initInstance();
-        connect(BitTorrent::Session::instance(), &BitTorrent::Session::restored, this, [this]()
-        {
-#ifndef DISABLE_WEBUI
-            m_webui = new WebUI(this);
-#ifdef DISABLE_GUI
-            if (m_webui->isErrored())
-                QCoreApplication::exit(1);
-            connect(m_webui, &WebUI::fatalError, this, []() { QCoreApplication::exit(1); });
+    BitTorrent::Session::initInstance();
+#ifndef DISABLE_GUI
+    UIThemeManager::initInstance();
 
-            const Preferences *pref = Preferences::instance();
+    m_desktopIntegration = new DesktopIntegration(this);
+    m_desktopIntegration->setToolTip(tr("Loading torrents..."));
+#ifndef Q_OS_MACOS
+    auto *desktopIntegrationMenu = new QMenu;
+    auto *actionExit = new QAction(tr("E&xit"), desktopIntegrationMenu);
+    actionExit->setIcon(UIThemeManager::instance()->getIcon(u"application-exit"_qs));
+    actionExit->setMenuRole(QAction::QuitRole);
+    actionExit->setShortcut(Qt::CTRL | Qt::Key_Q);
```

Comment:
```suggestion
    actionExit->setShortcut(QKeySequence::Quit);
```

Reply:
Is it the same?
Note that I just copied the key sequence from existing code in MainWindow.

Reply:
Not for Windows: https://doc.qt.io/qt-6/qkeysequence.html#standard-shortcuts
I suppose this is the reason of not using `QKeySequence::Quit` in MainWindow...
Forget this.